soc/arm/silabs_exx32/Kconfig

```diff
@@ -26,6 +26,11 @@ config SOC_PART_NUMBER
 	  that you should not set directly. The part number selection choice defines
 	  the default value for this string.
 
+config HAS_GECKO_DCDC
+	bool
+	help
+		Enable the DCDC for the SoC.
```

Comment:
Looks like the indentation is wrong here. Please indent one tab and two spaces.

I also think this config should be moved to `ext/hal/silabs/gecko/Kconfig`.